Gemfile

```diff
@@ -205,3 +205,5 @@ group :production do
   # https://newrelic.com/
   gem("newrelic_rpm")
 end
+
+gem "solid_queue", "~> 0.3.0"
```

Comment:
Two questions:
- Is that version lock intentional, or just the result of running `bundle add`
- Could we move this up to around line 45 (or wherever seems to make sense)? 

(I just went through the Gemfile last week to try to organize it a bit, and that helped me find a bunch of redundant and unnecessary gem declarations. I tried to group things sort of by functionality.)

Reply:
`bundle add` as you suspected and sure.